onnxruntime/python/tools/tensorrt/perf/build/Dockerfile.custom-trt-perf

```diff
@@ -0,0 +1,64 @@
+# --------------------------------------------------------------
```

Comment:
I think it would be nice to have a set of Dockerfiles (Ubuntu and CentOS) 
that could be shared/re-used by the various CI/packaging pipelines, perf and other scenarios? 
+@chilo-ms